cli/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Cli Debug",
```

Comment:
Let's call this `cli - attach`, both so that it matches the format of the other entries in the list and indicates that it's attaching to a thing rather than actually running a thing.

Reply:
fixed